plugins/ObjectsFormat.cc

```diff
@@ -283,6 +283,17 @@ void ObjectsFormat::ResetJetType(JetType& I) {
     I.dPhi_met    = -1.;
     I.dPhi_Jet1   = -1.;
     I.puId        = -1.;
+    I.ptGenJ      = -10.;
+    I.etaGenJ     = -4.;
+    I.phiGenJ     = -4.;
+    I.massGenJ    = -10.;
+    I.ptGen       = -10.;
+    I.etaGen      = -4.;
+    I.phiGen      = -4.;
+    I.massGen     = -10.;
+    I.ptLhe       = -10.;
+    I.etaLhe      = -4.;
+    I.phiLhe      = -4.;
```

Comment:
I don't see the inclusion of these variables (in their correct order) into the JetType struct [see line 325]. Please add.